standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -1603,21 +1645,24 @@ public void commitTxn(CommitTxnRequest rqst) throws NoSuchTxnException, TxnAbort
         }
 
         if (!isHiveReplTxn) {
-          createCommitNotificationEvent(dbConn, txnid , txnType);
+          createCommitNotificationEvent(jdbcTemplate, txnid , txnType);
         }
 
         LOG.debug("Going to commit");
-        dbConn.commit();
+        jdbcTemplate.commit();
 
         if (MetastoreConf.getBoolVar(conf, MetastoreConf.ConfVars.METASTORE_ACIDMETRICS_EXT_ON)) {
           Metrics.getOrCreateCounter(MetricsConstants.TOTAL_NUM_COMMITTED_TXNS).inc();
         }
       } catch (SQLException e) {
         LOG.debug("Going to rollback: ", e);
-        rollbackDBConn(dbConn);
+        jdbcTemplate.rollback();
```

Comment:
i don't get it, we did refactor in some methods, replaced retry with retryhandler and used jdbctemplatewrapper. here we just using jdbcTemplate to access the connection object?